buildSrc/build.gradle

```diff
@@ -9,5 +9,5 @@ repositories {
     }
 }
 dependencies {
-    implementation "edu.wpi.first:native-utils:2024.5.2"
+    implementation "edu.wpi.first:native-utils:2024.7.1"
```

Comment:
Bump this to 7.2 for 13.3

Reply:
(Which will be added in wpilibsuite/native-utils#198)